src/test/ui/borrowck/two-phase-nonrecv-autoref.rs

```diff
@@ -1,4 +1,4 @@
-// revisions: nll
+// revisions: base nll
 //[nll]compile-flags: -Z borrowck=mir
 
 //[g2p]compile-flags: -Z borrowck=mir -Z two-phase-beyond-autoref
```

Comment:
Does this option still exist in the compiler?

Reply:
I'm not sure - preexisting  thought and I'd rather not make this PR anything other than mechanical. (Though I should try to remember to come back to this at some point)